packages/plugin-prisma/src/generator.ts

```diff
@@ -42,22 +42,22 @@ generatorHandler({
   onGenerate: async (options) => {
     checkTSVersion();
     const config = options.generator.config as GeneratorConfig;
-    const prismaOutputLocation =
+    const prismaLocation =
       config.clientOutput ??
       options.otherGenerators.find((gen) => gen.provider.value === 'prisma-client')?.output
         ?.value ??
       options.otherGenerators.find((gen) => gen.provider.value === 'prisma-client-js')!.output!
         .value!;
 
-    const extensions = ['.js', '.ts', '.cjs', '.mjs', '.cts', '.mts'];
+    // const extensions = ['.js', '.ts', '.cjs', '.mjs', '.cts', '.mts'];
 
-    const prismaLocation =
-      prismaOutputLocation.startsWith('@') ||
-      extensions.some((ext) => prismaOutputLocation.endsWith(ext))
-        ? prismaOutputLocation
-        : prismaOutputLocation.startsWith('./')
-          ? `./${posix.join(prismaOutputLocation, 'index.js')}`
-          : posix.join(prismaOutputLocation, 'index.js');
+    // const prismaLocation =
+    //   prismaOutputLocation.startsWith('@') ||
+    //   extensions.some((ext) => prismaOutputLocation.endsWith(ext))
+    //     ? prismaOutputLocation
+    //     : prismaOutputLocation.startsWith('./')
+    //       ? `./${posix.join(prismaOutputLocation, 'index.js')}`
+    //       : posix.join(prismaOutputLocation, 'index.js');
```

Comment:
I think this is the fix for an issue in the latest prisma release.  It looks like prisma always generates a package.json file with an exports field defined.  I previously was importing from /prisma-path/index.ts instead of just prisma-path because I was worried about esm resolution, but with the package.json exports, I don't think thats actually an issue anymore